### ISSUE TITLE
Staging Review finding: Directional language in email alert 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#123438]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Horizon
**Product:** MHV on VA.gov
**Feature:** Notification email update prompts

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not align with VA.gov voice and tone content principles.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-aligns-with-vagov-voice-and-tone-content-principles)
**VA.gov Experience Standard - category:** [Credibility](https://design.va.gov/about/experience-standards/#category-credibility)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @kkitagawa-bosch (content), @jasonday (Accessibility)

### Description

The "confirm your contact email" alert uses directional language ("below") which conflicts with our content guidance. Not all users have a linear experience, so we avoid linear language. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="887" height="655" alt="Image" src="https://github.com/user-attachments/assets/1f757588-a5ff-4cca-8ddc-640c461d174d" />

</details>

## Recommended action

Change to "on this page" 

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Neutral language](https://design.va.gov/content-style-guide/neutral-language)
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
